### PR TITLE
Support short name of bundles in import command.

### PR DESCRIPTION
**Pull request by ndinh215**
I implemented the issue #64 . Please review it. Thanks.

### CODE REVIEW

**Comment by ndinh215** [comment]
Some hardcode made this bundle not able to run both with Symfony 2 and 3. For example:
```
$classes = [
'Symfony\Component\Validator\Validator' => '/Resources/translations',
'Symfony\Component\Form\Form' => '/Resources/translations',
'Symfony\Component\Security\Core\Exception\AuthenticationException' => '/../../Resources/translations',
];
```

It needs to be refactored. I will wait for your decision, @saimaz . Thanks.

File: Command/ImportCommand.php [hunk 1]
```diff
@@ -111,14 +112,27 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+        if ($isShortName) {
+            $bundle = $this->getApplication()->getKernel()->getBundle($bundleName);
+
+            if (!$bundle) {
+                throw new InvalidArgumentException(
+                    "Invalid bundle '{$bundleName}'"
+                );
+            }
+
+            return 1;
```
Comment (saimaz) [inline Command/ImportCommand.php]: Should be return `true`.

Comment (saimaz) [inline Command/ImportCommand.php]: btw this return true should be at the end of function after all checks are completed. IMO not inside the if.

Comment (ndinh215) [inline Command/ImportCommand.php]: Got it, thks @saimaz .

**Comment by ndinh215** [comment]
I fixed it, please check. Thanks.

**Comment by saimaz** [comment]
Sorry, I've missed that we are checking namespace. It's already checked by Symfony, no need to check it twice. Actually, we don't need validateBundle function. Just request it from the kernel, if bundle won't exist kernel will throw an exception.

**Comment by ndinh215** [comment]
@saimaz , you are correct. Without a bundle existence, it will throw the exception:
```
Bundle "AcmeTestBundle1" does not exist or it is not enabled. Maybe you forgot to add it in the registerBundles() method of your AppKernel.php file?

```
Thanks.

**Comment by ndinh215** [comment]
I removed it. Please check. Thanks.

**Pull request merged**